### PR TITLE
cli: add workflow create

### PR DESCRIPTION
**Pull request by diegodelemos**
* Updates RS OpenAPI spec.

Signed-off-by: Diego Rodriguez <diego.rodriguez@cern.ch>

### CODE REVIEW

File: reana_client/api/client.py [hunk 1]
```diff
@@ -93,17 +93,16 @@ def run_analysis(self, user, organization, reana_spec):
                                   organization=organization,
                                   reana_spec=json.loads(json.dumps(
                                       reana_spec, sort_keys=True))).result()
-
-            if http_response.status_code == 200:
+            if http_response.status_code == 201:
                 return response
             else:
                 raise Exception(
                     "Expected status code 200 but replied with "
```
Comment (hjhsalo) [inline reana_client/api/client.py]: Should the Exception message be `Expected status code 201 but <REANA_SERVER> replied with ...`?

File: reana_client/cli/workflow.py [hunk 2]
```diff
@@ -146,22 +146,22 @@ def workflow_create(ctx, file, user, organization, skip_validation):
 
     click.echo('Workflow `{}` has been created.'.format(get_random_name()))
```
Comment (hjhsalo) [inline reana_client/cli/workflow.py]: Should this be changed and moved to end (of this function) or commented out for now?

File: reana_client/api/client.py [hunk 3]
```diff
@@ -84,7 +84,7 @@ def get_all_analyses(self):
         except Exception:
             raise
 
-    def run_analysis(self, user, organization, reana_spec):
+    def create_workflow(self, user, organization, reana_spec):
         """Create an analysis."""
```
Comment (hjhsalo) [inline reana_client/api/client.py]: Should be `Create a workflow` in the docstring?

**Comment by diegodelemos** [comment]
ready

**Pull request merged**